### PR TITLE
Xrddev 875 localgroups members e2etests

### PR DESCRIPTION
**Pull request by olliru**
Added local group member e2e tests

### CODE REVIEW

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: This kind of bootstrapping repeats a lot, could this be implemented as e.g. a helper method?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+    // Change description
+    localGroupPopup.changeDescription('');
+    localGroupPopup.clickHeader();
+    browser.assert.containsText('//div[contains(@class, "v-snack__content")]', 'Request failed with status code 500');
+    mainPage.closeSnackbar();
+    localGroupPopup.close();
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")] and .//*[contains(text(), "Group4")]]')
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+    localGroupPopup.changeDescription(browser.globals.test_string_300.slice(0,256));
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Would maybe be beneficial to test corner case: 255 characters long description _can_ be saved. Even though acceptance criteria does not require it.

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+
+    // Open TestGov Internal Servers and local group details view
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openTestService();
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    clientLocalGroups.openBacDetails();
+    browser.waitForElementVisible(localGroupPopup);
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Maybe even this part, also?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    // Delete and cancel
+    localGroupPopup.deleteThisGroup();
+    browser.waitForElementVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Delete group?")]');
+    localGroupPopup.cancelMemberRemove();
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: This cancels _local group removal_, not _local group member removal_, right? Should the name be different?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+    localGroupPopup.close();
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")]]')
+
+    // Delete and confirm
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    localGroupPopup.deleteThisGroup();
+    browser.waitForElementVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Delete group?")]');
+    localGroupPopup.confirmMemberRemove();
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: This confirms _local group removal_, not _local group member removal_, right? Should the name be different?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+    browser.waitForElementVisible('//table[contains(@class, "details-certificates")]//tr[.//*[contains(text(),"cbb")]]')
+
+    // Delete and confirm
+    clientLocalGroups.openCbbDetails();
+    browser.waitForElementVisible(localGroupPopup);
+
+    localGroupPopup.deleteThisGroup();
+    browser.waitForElementVisible('//*[contains(@data-test, "dialog-title") and contains(text(), "Delete group?")]');
+    localGroupPopup.confirmMemberRemove();  
+    browser.waitForElementVisible(clientLocalGroups);
```
Comment (jansu76) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Should this test that the removed local group was successfully removed, ie. does not exist in the list anymore?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js (hunk 1 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Implemented

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js (hunk 2 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Implemented

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js (hunk 4 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Implemented

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js (hunk 5 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Implemented

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js (hunk 6 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: Implemented

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js (hunk 3 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js]: The way how tests navigate to the actual target area of the SUT is going to be changed when the page model is refactored, so I'm not changing it at this time.
